src/app/shared/components/switcher/switcher.component.html

```diff
@@ -0,0 +1,12 @@
+<!-- @format -->
+<div>
+  <label class="switch">
+    <input
+      #switch
+      (change)="change(data)"
+      [checked]="isChecked"
+      type="checkbox"
+    />
+    <span class="slider"></span>
+  </label>
+</div>
```

Comment:
I didn't realize we'd use a new component to make a slider like this. Interesting!